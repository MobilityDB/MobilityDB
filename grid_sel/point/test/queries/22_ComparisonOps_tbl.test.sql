-------------------------------------------------------------------------------

SELECT count(*) FROM tbl_tgeompoint, tbl_geompoint WHERE g #= temp IS NOT NULL;
SELECT count(*) FROM tbl_tgeogpoint, tbl_geogpoint WHERE g #= temp IS NOT NULL;

SELECT count(*) FROM tbl_tgeompoint t1, tbl_tgeompoint t2 WHERE t1.temp #= t2.temp IS NOT NULL;
SELECT count(*) FROM tbl_tgeogpoint t1, tbl_tgeogpoint t2 WHERE t1.temp #= t2.temp IS NOT NULL;

SELECT count(*) FROM tbl_tgeompoint3D, tbl_geompoint3D WHERE g #= temp IS NOT NULL;
SELECT count(*) FROM tbl_tgeogpoint3D, tbl_geogpoint3D WHERE g #= temp IS NOT NULL;

SELECT count(*) FROM tbl_tgeompoint3D t1, tbl_tgeompoint3D t2 WHERE t1.temp #= t2.temp IS NOT NULL;
SELECT count(*) FROM tbl_tgeogpoint3D t1, tbl_tgeogpoint3D t2 WHERE t1.temp #= t2.temp IS NOT NULL;

-------------------------------------------------------------------------------

SELECT count(*) FROM tbl_tgeompoint, tbl_geompoint WHERE g #<> temp IS NOT NULL;
SELECT count(*) FROM tbl_tgeogpoint, tbl_geogpoint WHERE g #<> temp IS NOT NULL;

SELECT count(*) FROM tbl_tgeompoint t1, tbl_tgeompoint t2 WHERE t1.temp #<> t2.temp IS NOT NULL;
SELECT count(*) FROM tbl_tgeogpoint t1, tbl_tgeogpoint t2 WHERE t1.temp #<> t2.temp IS NOT NULL;

SELECT count(*) FROM tbl_tgeompoint3D, tbl_geompoint3D WHERE g #<> temp IS NOT NULL;
SELECT count(*) FROM tbl_tgeogpoint3D, tbl_geogpoint3D WHERE g #<> temp IS NOT NULL;

SELECT count(*) FROM tbl_tgeompoint3D t1, tbl_tgeompoint3D t2 WHERE t1.temp #<> t2.temp IS NOT NULL;
SELECT count(*) FROM tbl_tgeogpoint3D t1, tbl_tgeogpoint3D t2 WHERE t1.temp #<> t2.temp IS NOT NULL;

-------------------------------------------------------------------------------
