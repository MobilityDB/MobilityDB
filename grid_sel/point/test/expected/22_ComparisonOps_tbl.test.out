SELECT count(*) FROM tbl_tgeompoint, tbl_geompoint WHERE g #= temp IS NOT NULL;
 count 
-------
 10000
(1 row)

SELECT count(*) FROM tbl_tgeogpoint, tbl_geogpoint WHERE g #= temp IS NOT NULL;
 count 
-------
 10000
(1 row)

SELECT count(*) FROM tbl_tgeompoint t1, tbl_tgeompoint t2 WHERE t1.temp #= t2.temp IS NOT NULL;
 count 
-------
   679
(1 row)

SELECT count(*) FROM tbl_tgeogpoint t1, tbl_tgeogpoint t2 WHERE t1.temp #= t2.temp IS NOT NULL;
 count 
-------
   675
(1 row)

SELECT count(*) FROM tbl_tgeompoint3D, tbl_geompoint3D WHERE g #= temp IS NOT NULL;
 count 
-------
 10000
(1 row)

SELECT count(*) FROM tbl_tgeogpoint3D, tbl_geogpoint3D WHERE g #= temp IS NOT NULL;
 count 
-------
 10000
(1 row)

SELECT count(*) FROM tbl_tgeompoint3D t1, tbl_tgeompoint3D t2 WHERE t1.temp #= t2.temp IS NOT NULL;
 count 
-------
   677
(1 row)

SELECT count(*) FROM tbl_tgeogpoint3D t1, tbl_tgeogpoint3D t2 WHERE t1.temp #= t2.temp IS NOT NULL;
 count 
-------
   677
(1 row)

SELECT count(*) FROM tbl_tgeompoint, tbl_geompoint WHERE g #<> temp IS NOT NULL;
 count 
-------
 10000
(1 row)

SELECT count(*) FROM tbl_tgeogpoint, tbl_geogpoint WHERE g #<> temp IS NOT NULL;
 count 
-------
 10000
(1 row)

SELECT count(*) FROM tbl_tgeompoint t1, tbl_tgeompoint t2 WHERE t1.temp #<> t2.temp IS NOT NULL;
 count 
-------
   679
(1 row)

SELECT count(*) FROM tbl_tgeogpoint t1, tbl_tgeogpoint t2 WHERE t1.temp #<> t2.temp IS NOT NULL;
 count 
-------
   675
(1 row)

SELECT count(*) FROM tbl_tgeompoint3D, tbl_geompoint3D WHERE g #<> temp IS NOT NULL;
 count 
-------
 10000
(1 row)

SELECT count(*) FROM tbl_tgeogpoint3D, tbl_geogpoint3D WHERE g #<> temp IS NOT NULL;
 count 
-------
 10000
(1 row)

SELECT count(*) FROM tbl_tgeompoint3D t1, tbl_tgeompoint3D t2 WHERE t1.temp #<> t2.temp IS NOT NULL;
 count 
-------
   677
(1 row)

SELECT count(*) FROM tbl_tgeogpoint3D t1, tbl_tgeogpoint3D t2 WHERE t1.temp #<> t2.temp IS NOT NULL;
 count 
-------
   677
(1 row)

